recipes/fenics-oasis/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "fenics-oasis" %}
+{% set version = "2019.1" %}
+{% set python_min = "3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mikaem/Oasis/archive/{{ version }}.tar.gz
+  sha256: 4456d5f042292e30bffda0487b57e282fe6e776cb6cdc832140072983b2c9d8d
+
+build:
+  entry_points:
+    - oasis=oasis.run_oasis:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]  # Exclude Windows builds
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python
+    - fenics
+    - numpy
+    - matplotlib-base
+    - procps-ng  # [linux]
+    - wget  # [linux]
+    - curl  # [osx]
```

Comment:
Are these really needed? We prefer noarch package, if they are really mandatory dependencies you should try OS-noarch: https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

Reply:
@ocefpaf I have removed `wget` and `curl` from the runtime dependencies and restored `noarch: python`.

These tools were only used to fetch mesh files for example problems, which are not essential for the core functionality of `fenics-oasis`.

Thanks for the review!